README.md

```diff
@@ -1 +1,126 @@
+Clicking on a specific result provides users with images and detailed descriptions of that particular location.
+Users can leave feedback on whether the parking spot was accessible.
+
+
+
+## Usage
+
+How to Use:
+
+    - Search for Parking Spots
+    Open the application and input your desired location in the search bar.
+    Explore the map to discover available parking spots near the searched location.
+
+    - Filter Results
+    After receiving search results, utilize the provided filters to narrow down parking options based on specific criteria.
+   
+    - View Search History
+    Access your search history by clicking on 'View Previous Searches' to review past results and locations.
+   
+    - Display Detailed Information
+    Click on a specific parking spot to access a wealth of information, including street view, the weather in that location comprehensive descriptions, and user feedback.
+
+## Credits
+
+Please see the link to all our collaborators GitHub profiles:
+
+Milo: https://github.com/miloyang
+Felix: https://github.com/honguyen00
+Marion: https://github.com/MFMcNolt
+
+## Features
+
+1. Search Feature
+User Story
+As a person with a car, I want to be able to search for a destination and view parking spots near it, so that I can plan my trip better.
+
+Acceptance Criteria
+The application opens with an intuitive search interface.
+Users can enter their desired location, triggering a map display with relevant parking results.
+Interaction with the map is smooth, providing a seamless experience for planning trips.
+
+2. Filter Feature
+User Story
+As a person with special needs, I want to be able to filter parking spots so that I can find a spot that matches my requirements.
+
+Acceptance Criteria
+Search results are presented in a user-friendly interface.
+Users have the ability to apply various filters, including radius, fees, and disabled parking availability.
+Filters are designed to accommodate diverse accessibility requirements, ensuring personalized results.
+
+3. Search History Feature
+User Story
+As a person using the app, I want to see my search history so that I can know where I have searched previously.
+
+Acceptance Criteria
+A 'View Previous Searches' feature enables users to access and review their search history.
+Previous search results are conveniently organized, providing users with a quick reference to their past parking explorations.
+
+4. Display Description Feature
+User Story
+As a person with a car, I want to select from a filtered search result, so that I can see feedback, images, and descriptions of parking facilities and the weather at that parking facility.
+
+Acceptance Criteria
+Clicking on a specific parking spot in the search results opens a detailed view.
+Users are presented with images showcasing the parking facility, enhancing the visual understanding.
+Comprehensive descriptions and user feedback provide valuable insights into the accessibility and suitability of the parking spot.
```

Comment:
same issues, I don't think we need to include all of these. Keep it short and simple with:
-- Description
-- Installation
-- Usage 
-- Features
-- Credit
-- Technology used